stac_validator.py

```diff
@@ -126,69 +112,58 @@ def fetch_specs(self, version):
 
         return ITEM_SCHEMA, ITEM_GEOJSON_SCHEMA, CATALOG_SCHEMA
 
-    def validate_stac(self, stac_file, schema):
+    def validate_json(self, stac_content, schema):
         """
         Validate stac
-        :param stac_file: input stac_file
+        :param stac_content: input stac file content
         :param schema of STAC (item, catalog)
         :return: validation message
         """
 
         stac_schema = json.loads(schema)
         try:
-            validate(stac_file, stac_schema)
-            self.message["valid_stac"] = True
+            validate(stac_content, stac_schema)
+            return True, None
         except RefResolutionError as error:
             # See https://github.com/Julian/jsonschema/issues/362
             # See https://github.com/Julian/jsonschema/issues/313
             # See https://github.com/Julian/jsonschema/issues/98
             try:
                 self.geojson_resolver = RefResolver(
-                    base_uri="file://{}/".format(cache["geojson_resolver"]), referrer="geojson.json"
+                    base_uri="file://{}/".format(cache["geojson_resolver"]),
+                    referrer="geojson.json"
                 )
-                validate(stac_file, stac_schema, resolver=self.geojson_resolver)
-                self.message["valid_stac"] = True
+                validate(stac_content, stac_schema, resolver=self.geojson_resolver)
+                return True, None
             except Exception as error:
-                self.message["valid_stac"] = False
-                self.message["error_message"] = f"{error.args}"
+                return False, f"{error.args}"
         except ValidationError as error:
-            self.message["valid_stac"] = False
-            self.message["error_message"] = f"{error.message} of {list(error.path)}"
-
+            # TODO: log error
+            return False, f"{error.message} of {list(error.path)}"
```

Comment:
`validate_stac` now `validate_json` returns two things:
- True or False tag
- error message

Reply:
Why the change to `validate_json`?
I like the True/False tags and error message